docs/source/markdown/options/network.md

```diff
@@ -62,11 +60,20 @@ Valid _mode_ values are:
     container to host using the gateway address. The latter can be overridden
     by passing **--map-gw** in the pasta-specific options (despite not being an
     actual pasta(1) option). \
+    For better integration with DNS handling, **--dns-forward 169.254.1.1** is passed,
+    and this address is added to resolv.conf(5) as first resolver. It is possible to pass
+    **--dns-forward** explicitly in case a different IP address should be used.
+    To make the `host.containers.internal` /etc/hosts entry work and allow connections
+    to the host, **--map-guest-addr 169.254.1.2** is passed. Again, it can be set
+    explicitly to choose a different IP address. \
     Also, **-t none** and **-u none** are passed if, respectively, no TCP or
-    UDP port forwarding from host to container is configured, to disable
-    automatic port forwarding based on bound ports. Similarly, **-T none** and
-    **-U none** are given to disable the same functionality from container to
+    UDP port forwarding from host to container is configured (via Podman's
+    **--publish** or by passing the pasta **-t**/**-u** options directly as options),
```

Comment:
`passing pasta's **-t**/**-u** options directly),`

...but not a strong preference, I see why might want to specify "options as options", even though it feels redundant and not necessarily clearer to me.

Reply:
dropped